tests/test_filtering.py

```diff
@@ -62,6 +64,13 @@ class WidgetListView(GenericModelView):
 
         filtering = Filtering(
             color=operator.eq,
```

Comment:
eg `Filter(operator.eq)`, `Filter(operator.eq, 'size')`, `Filter(operator.eq, field='size' empty='....')` and so on